DependencyInjection/FactoryConfiguration.php

```diff
@@ -19,18 +19,24 @@ class FactoryConfiguration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
+        if(method_exists(TreeBuilder::class, 'getRootNode')) {
```

Comment:
Please respect the PSR 2 coding style :
```php
if (condition) {
    // ...
} else {
    // ...
}
```

Reply:
Also, can you add a small comment indicating that this check is covering the deprecation of the treebuilder v5 please ?

Reply:
Done! Thank you too.